aiida_restapi/models.py

```diff
@@ -129,3 +130,26 @@ class Group(AiidaModel):
     type_string: Optional[str] = Field(description="Type of the group")
     user_id: Optional[str] = Field(description="Id of the user that created the node.")
     description: Optional[str] = Field(description="Short description of the group.")
+
+
+class Process(AiidaModel):
+    """AiiDA Process Model"""
+
+    _orm_entity = orm.ProcessNode
+
+    id: Optional[int] = Field(description="Unique id (pk)")
+    uuid: Optional[UUID] = Field(description="Unique uuid")
```

Comment:
UUID stands for "universally unique identifier", i.e. "unique" is already part of the abbreviation